scripts/utils/resolveinventoryitemassets.lua

```diff
@@ -7,6 +7,10 @@ return function(prefab)
   local atlas = GetInventoryItemAtlas(item_tex)
   local localized_name = STRINGS.NAMES[string.upper(prefab)] or prefab
   local prefabData = Prefabs[prefab]
+  
+  local registered_image, registered_altas = GetFoodAtlas(prefab)
+  item_tex = registered_image or item_tex
+  atlas = registered_altas or atlas
 
   if prefabData then
     -- first run we find assets with exact match of prefab name
```

Comment:
Hmm, so for what reason the original way of providing images doesn't work?
Normally, if you provide Assets section in the prefab file with INV_IMAGE / ATLAS - they are used for inventory item. You said something about using SetPrefabNameOverride. Wouldn't it just work if you also copied the assets for the copied prefabs?

Reply:
Except winterfeast don't have explicit atlases :)
Hate that part about some of the klei's api

Reply:
In my case I didn't provide any INV_IMAGE or ATLAS.

Reply:
The dynamic parser is what Display Food Values use, spawn and remove to see its real image and atlas from inst.replica.inventoryitem.